sw/device/lib/testing/test_framework/BUILD

```diff
@@ -67,46 +119,49 @@ cc_library(
     ],
     target_compatible_with = [OPENTITAN_CPU],
     deps = [
-        ":linker_script",
         "//sw/device/lib/base:csr",
         "//sw/device/lib/base:macros",
         "//sw/device/lib/crt",
         "//sw/device/lib/runtime:hart",
         "//sw/device/lib/runtime:log",
-        "//sw/device/silicon_creator/lib:manifest_size",
     ],
 )
 
 cc_library(
-    name = "ottf_test_config",
-    hdrs = [
-        "ottf_test_config.h",
+    name = "ottf_start_silicon_creator_slot_a",
```

Comment:
Hmm, I wasn't expecting to see 3 different `ld_libraries` and 3 different `cc_libraries` for this purpose -- maybe I'm missing something but I was expecting the slot_a/slot_b/virtual_addr choice to be a parameter for the flash test binary.

Added to our doc to talk about during our next 1:1.

Reply:
This is just a stepping stone to getting the the proper parameter (e.g., something like "slot") added to the `opentitan_functest`/`opentitan_flash_binary` macros and to fix #14210 in the interim. See #10712 for the remaining steps I am implementing, note step 3 is updating the bazel rules/macros to clean things up.

Even with the macros updated, we still need 3 different `ld_libraries` for the bazel macros to choose from to build the proper OT flash binary (and we will need three more to allow building for the silicon_owner stage), unless I am missing something. Unless you were thinking generating these on the fly? (though that may get confusing to follow) I'm not sure I follow the alternative you were thinking here? But SG we can discuss later this week.